### ISSUE TITLE
Why do we need to download the CLIP model separately?

### ISSUE THREAD

**Issue by XavierXiao** [TencentARC/T2I-Adapter#11]
In the readme, you mentioned that we need to download the weights of CLIP, but isn't it included in the SD v1.4 checkpoint?

**Comment by MC-E** [comment]
Of course, you can comment out the path to CLIP in the config file, and it will be downloaded automatically.
# params:
#     version: models/clip-vit-large-patch14
